File: src/test/java/com/developersam/game/ten/TenPerformanceTest.java
package com.developersam.game.ten;

import org.junit.Test;

/**
 * Run a performance test on TEN.
 */
public final class TenPerformanceTest {
    
    /**
     * Test the performance of the game by run a game between two MCTS AI.
     */
    @Test
    public void testPerformance() {
        Board.runAGameBetweenTwoAIs(50);
    }
    
}

File: src/test/kotlin/com/developersam/game/ten/TenPerformanceTest.kt
package com.developersam.game.ten

import org.junit.Test

/**
 * Run a performance test on TEN.
 */
class TenPerformanceTest {

    /**
     * Test the performance of the game by run a game between two MCTS AI.
     */
    @Test
    fun testPerformance() {
        Board.runAGameBetweenTwoAIs(
                aiThinkingTime = 50, printGameStatus = false)
    }

}